stix2elevator/convert_cybox.py

```diff
@@ -486,6 +586,16 @@ def convert_cybox_object(obj):
     elif isinstance(prop, NetworkConnection):
         # potentially returns multiple objects
         objs = convert_network_connection(prop)
+    elif isinstance(prop, HTTPSession):
+        objs[0] = convert_http_session(prop)
+    elif isinstance(prop, NetworkPacket):
+        objs[0] = convert_network_packet(prop)
+    elif isinstance(prop, NetworkSocket):
+        objs[0] = convert_network_socket(prop)
+    # elif isinstance(prop, HTTPRequestResponse):
```

Comment:
Do you think it would be helpful to have a warning here, or at least a comment that we don't support HTTPRequestResponse unless it is within a HTTPSession?

Reply:
I'm not sure that is the best idea, since there are probably many similar cases for which we wouldn't have a message. On the other hand, I will remove the commented out code.